docs/source/index.rst

```diff
@@ -9,11 +9,13 @@ Benchmarks
 Kubestone currently supports the following benchmarks:
 
 * Disk
-   - fio
+   - `fio`_
```

Comment:
This link was in my original commit, but then I removed it, because I think it would be better to link our own fio/iperf documentation. What do you think?

Reply:
I mean: https://kubestone.readthedocs.io/en/latest/benchmarks/disk.html#fio

Reply:
I second this!